src/addons/Portal/Portal.js

```diff
@@ -37,15 +37,27 @@ class Portal extends Component {
     /** Controls whether or not the portal should close on blur of the trigger. */
     closeOnTriggerBlur: PropTypes.bool,
 
-    /** Controls whether or not the portal should close when mousing out of the trigger. */
-    closeOnTriggerMouseLeave: PropTypes.bool,
+    /** Controls whether or not the portal should close on blur of the trigger. */
+    closeOnTriggerClick: PropTypes.bool,
+
+    /**
+     * Controls whether or not the portal should close when mousing out of the
+     * trigger OR the portal content.
+     */
+    closeOnMouseLeave: PropTypes.bool,
```

Comment:
That was my initial thought, but then I was wondering how do you configure allowing mousing from the trigger to the portal? I think I just figured it out though:
- If only `closeOnTriggerMouseLeave` is set, then mouseover/leave on the portal won't do anything
- If `closeOnPortalMouseLeave` is set, then mouseover of the portal clears the timer, mouseleave restarts the timer.
So `closeOnPortalMouseLeave` has the added effect of preventing close when mousing over the gap from trigger -> portal

This actually lets us keep more closely with SUI's hoverable:
- `hoverable => false`  is `{ closeOnTriggerMouseLeave: true }`
- `hoverable => true`  is `{ closeOnTriggerMouseLeave: true, closeOnPortalMouseLeave: true }`